src/screens/ImagePicker/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from 'react';
+import { Platform, Alert, View, ActivityIndicator, Text } from 'react-native';
+import { connect } from 'react-redux';
+import { CommonActions } from '@react-navigation/native';
+import * as ImagePicker from 'expo-image-picker';
+import * as FileSystem from 'expo-file-system';
+import * as ImageManipulator from 'expo-image-manipulator';
+import mime from 'mime';
+import Users from '../../api/users';
+import { UserType } from '../../types/UserType';
+import { ActionType, ActionWithoutPayloadType } from '../../types/store/StoreType';
+import MainActions from '../../store/main/actions';
+import { navigate } from '../../navigationRef';
+import commonStyle from '../../styles/common';
+import { GREY } from '../../styles/colors';
+import styles from './styles';
+import { NavigationType } from '../../types/NavigationType';
+import { IMAGE_MAX_SIZE } from '../../core/data/constants';
+
+interface ImagePickerContainerProps {
+  navigation: NavigationType;
+  loggedUser: UserType,
+  setLoggedUser: (user: UserType) => void,
+
+}
+
+const ImagePickerContainer = ({ navigation, loggedUser, setLoggedUser }:ImagePickerContainerProps) => {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    async function requestPermissions() {
+      if (Platform.OS !== 'web') {
+        const { status } = await ImagePicker.requestCameraRollPermissionsAsync();
+        if (status !== 'granted') {
+          Alert.alert('Sorry, we need camera roll permissions to make this work!');
```

Comment:
j'ai un comportement tres etrange ici : 
des que je clique sur "Ajouter une photo" l'alerte _'Sorry, we need camera roll permissions to make this work!'_ apparait directement puis par dessus juste apres ma galerie apparait en tout petit san que j'ai le temps de faire quoi que ce soit.

je peux choisir ma photo / la repositionner et enregistrer. lalerte est toujours la - je clique sur OK et ma photo est enregistrée
![IMG_2885](https://user-images.githubusercontent.com/24521863/102459903-88af2480-4046-11eb-9ecd-6ff7fb68589d.PNG)

Reply:
dis-moi si c'est toujours le cas après les modifs, c'est très bizarre en effet ^^